airbyte-integrations/bases/base-java/Dockerfile

```
@@ -1,5 +1,5 @@
 ARG JDK_VERSION=17.0.1
-FROM openjdk:${JDK_VERSION}-slim
+FROM openjdk:17.0.1-slim
```

```suggestion
FROM openjdk:${JDK_VERSION}-slim
```

let's just change this back to how it originally was